### PR TITLE
Add #delete to GoodData::AdsOutputStage

### PR DESCRIPTION
**Pull request by panjan**

### CODE REVIEW

**Comment by korczis** [comment]
LGTM

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-rubocop-zuul/296/ : FAILURE in 32s
- https://ci.intgdc.com/job/gooddata-ruby-tests-zuul/231/ : FAILURE in 14s
- https://ci.intgdc.com/job/gooddata-lcm-zuul/23/ : FAILURE in 1m 33s (non-voting)

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-rubocop-zuul/297/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-ruby-tests-zuul/232/ : FAILURE in 40m 37s
- https://ci.intgdc.com/job/gooddata-lcm-zuul/24/ : FAILURE in 1m 27s (non-voting)

File: spec/integration/ads_output_stage_spec.rb [hunk 1]
```diff
@@ -9,9 +9,9 @@
 describe GoodData::AdsOutputStage, :constraint => 'slow' do
   before(:all) do
     @client = ConnectionHelper.create_default_connection
-    # @ads = GoodData::DataWarehouse.create(client: @client, title: 'Test ADS', auth_token: ConnectionHelper::GD_PROJECT_TOKEN, environment: ProjectHelper::ENVIRONMENT)
+    @ads = GoodData::DataWarehouse.create(client: @client, title: 'Test ADS', auth_token: ConnectionHelper::GD_PROJECT_TOKEN, environment: ProjectHelper::ENVIRONMENT)
 
-    # try to delete all posible ads
+    try to delete all posible ads
```
Comment (vhtien) [inline spec/integration/ads_output_stage_spec.rb]: This line is a comment line.

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-rubocop-zuul/299/ : SUCCESS in 6s
- https://ci.intgdc.com/job/gooddata-ruby-tests-zuul/234/ : FAILURE in 40m 15s
- https://ci.intgdc.com/job/gooddata-lcm-zuul/26/ : FAILURE in 1m 42s (non-voting)

**Comment by panjan** [comment]
ok to test

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-rubocop-zuul/301/ : SUCCESS in 10s
- https://ci.intgdc.com/job/gooddata-ruby-tests-zuul/236/ : FAILURE in 42m 06s
- https://ci.intgdc.com/job/gooddata-lcm-zuul/28/ : FAILURE in 1m 38s (non-voting)

**Comment by panjan** [comment]
ok to test

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-rubocop-zuul/302/ : SUCCESS in 28s
- https://ci.intgdc.com/job/gooddata-ruby-tests-zuul/237/ : FAILURE in 42m 28s
- https://ci.intgdc.com/job/gooddata-lcm-zuul/29/ : SUCCESS in 2m 29s (non-voting)

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-rubocop-zuul/304/ : SUCCESS in 11s
- https://ci.intgdc.com/job/gooddata-ruby-tests-zuul/239/ : FAILURE in 2m 00s
- https://ci.intgdc.com/job/gooddata-lcm-zuul/31/ : SUCCESS in 5m 49s (non-voting)

**Comment by panjan** [comment]
ok to test

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-rubocop-zuul/315/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-ruby-tests-zuul/250/ : SUCCESS in 42m 17s
- https://ci.intgdc.com/job/gooddata-lcm-zuul/42/ : SUCCESS in 2m 54s (non-voting)

**Pull request merged**